src/OpenTelemetry.Shims.OpenTracing/TracerShim.cs

```diff
@@ -52,7 +52,7 @@ private TracerShim(Trace.ITracer tracer)
                 throw new ArgumentNullException(nameof(format));
             }
 
-            if (carrier == default)

```

Comment:
VS complained about this 🤷‍♂

Reply:
It's a bug (only in internal builds) being fixed.

Reply:
I ended up changing those lines to carrier == null (for now).